### DISCUSSION TITLE
Tooltips Typos

### DISCUSSION THREAD

**Discussion by andybrice** [supermerill/SuperSlicer#1322] [Q&A]
I've noticed a number of typographical errors in the Print Settings tooltips. I'd be happy to go through and fix these if you want.

Is it sufficient simply to edit [PrintConfig.cpp](https://github.com/supermerill/SuperSlicer/blob/master/src/libslic3r/PrintConfig.cpp)? Or would I also have to edit the localization files?

And should I still create a new branch for such small changes?

**Comment by supermerill** [comment]
> Is it sufficient simply to edit PrintConfig.cpp? Or would I also have to edit the localization files?

yes, that will create an automatic pull request.
